package.json

```diff
@@ -65,7 +65,6 @@
     "webpack-bundle-analyzer": "^4.7.0"
   },
   "engines": {
-    "node": ">= 17",
```

Comment:
To morda ni več potrebno po merganju:
- https://github.com/sledilnik/zdravniki/pull/293